app/src/main/kotlin/org/eurofurence/connavigator/util/Formatter.kt

```diff
@@ -1,20 +1,22 @@
 package org.eurofurence.connavigator.util
 
+import android.content.Context
 import io.swagger.client.model.DealerRecord
 import io.swagger.client.model.EventRecord
+import org.eurofurence.connavigator.R
 import java.util.*
 
 /**
  * Created by David on 11-5-2016.
  */
 object Formatter {
```

Comment:
The formatter class is really old and shouldn't be used anymore, Maybe a new issue can be created where this class is instead replaced by extension functions?